src/components/Notifications.js

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash/fp'
+import { createRef } from 'react'
+import { h } from 'react-hyperscript-helpers'
+import ReactNotification from 'react-notifications-component'
+import { Component } from 'src/libs/wrapped-components'
+import 'react-notifications-component/dist/theme.css'
+import 'animate.css'
+
+
+let notificationsInstance
```

Comment:
I think it would be possible to just create the ref itself here, so `pushNotification` could access it directly, rather than having to go through the container instance.